src/mame/mame.lst

```diff
@@ -33107,6 +33107,7 @@ stratvox                        // [1980 Sun Electronics] Taito
 stratvoxa                       // [1980 Sun Electronics] Taito
 stratvoxb                       // bootleg
 ttmahjng                        // Taito
+jongpute                        // Alpha Denshi Co.
```

Comment:
Colors negatively affects gameplay, like some tiles are shown as white dragon but they aren't.
This should be marked MNW.

Reply:
http://mamedev.emulab.it/kale/fast/files/0001_6987612186.png